.github/workflows/code-coverage.yaml

```
@@ -0,0 +1,188 @@
+on:
+  workflow_call:
+    inputs:
+      run:
+        type: boolean
+        default: true
+      schemas:
+        type: string
+        default: ''
+      backend_skip:
```

GitHub Actions use `*_exclude` (`paths_exclude`, `branch_exclude`). For consistency, I would suggest changing `backend_skip` to align with this
```suggestion
      backend_exclude:
```